src/bridge/IInbox.sol

```diff
@@ -112,67 +57,6 @@ interface IInbox is IDelayedMessageProvider {
         view
         returns (uint256);
 
-    /**
-     * @notice Deposit eth from L1 to L2 to address of the sender if sender is an EOA, and to its aliased address if the sender is a contract
-     * @dev This does not trigger the fallback function when receiving in the L2 side.
-     *      Look into retryable tickets if you are interested in this functionality.
-     * @dev This function should not be called inside contract constructors
-     */
-    function depositEth() external payable returns (uint256);
-
-    /**
-     * @notice Put a message in the L2 inbox that can be reexecuted for some fixed amount of time if it reverts
-     * @dev all msg.value will deposited to callValueRefundAddress on L2
-     * @dev Gas limit and maxFeePerGas should not be set to 1 as that is used to trigger the RetryableData error
-     * @param to destination L2 contract address
-     * @param l2CallValue call value for retryable L2 message
-     * @param maxSubmissionCost Max gas deducted from user's L2 balance to cover base submission fee
-     * @param excessFeeRefundAddress gasLimit x maxFeePerGas - execution cost gets credited here on L2 balance
-     * @param callValueRefundAddress l2Callvalue gets credited here on L2 if retryable txn times out or gets cancelled
-     * @param gasLimit Max gas deducted from user's L2 balance to cover L2 execution. Should not be set to 1 (magic value used to trigger the RetryableData error)
-     * @param maxFeePerGas price bid for L2 execution. Should not be set to 1 (magic value used to trigger the RetryableData error)
-     * @param data ABI encoded data of L2 message
-     * @return unique message number of the retryable transaction
-     */
-    function createRetryableTicket(
```

Comment:
found it a bit annoying that this changed the inbox interface, can we have IInbox as the eth inbox, and make sth like IInboxBase e.g.
IEthInbox -> IInbox
IInbox -> IInboxBase

Reply:
right,  makes sense to preserve IInbox compatibility. updated